resources/zib/zib-HealthcareProvider-Organization.xml

```diff
@@ -105,7 +105,7 @@
       <sliceName value="organizationType" />
       <short value="OrganizationType" />
       <definition value="The type of healthcare provider, such as general hospital, or nursing home. If this field is filled in and an AGB code is used for the HealthcareProviderIdentificationNumber, the type must match the type implicitly contained in the AGB code." />
-      <comment value="The ValueSet binding strenght deviates from the zib because the binding strenght must be set to required in order to make the discriminator work. However, the open slice definition allows to extend codes." />

```

Comment:
Je kan je afvragen of dit commentaar uberhaupt wel zinvol is.

Reply:
Zie hierboven.